custom_components/tuya_local/devices/dry_pury_evo_dehumidifier.yaml

```diff
@@ -0,0 +1,160 @@
+name: Argo DRY PURY EVO WF dehumidifier
```

Comment:
Just use "Dehumidifer" for this top level name, in case the same device is sold under different brands elsewhere.